commit 8715ba45d0087f3cb67e85819ea5d4fdf7a86c88
Author: Jurica Vukadin <jurica.vukadin@rt-rk.com>
Date:   Thu Feb 7 16:29:37 2019 +0100

    ALSA: hda - Add quirk for HP EliteBook 840 G5
    
    commit 4cd3016ce996494f78fdfd87ea35c8ca5d0b413e upstream.
    
    This enables mute LED support and fixes switching jacks when the laptop
    is docked.
    
    Signed-off-by: Jurica Vukadin <jurica.vukadin@rt-rk.com>
    Cc: <stable@vger.kernel.org>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/sound/pci/hda/patch_conexant.c b/sound/pci/hda/patch_conexant.c
index 31a84a5a1338..fead0acb29f7 100644
--- a/sound/pci/hda/patch_conexant.c
+++ b/sound/pci/hda/patch_conexant.c
@@ -924,6 +924,7 @@ static const struct snd_pci_quirk cxt5066_fixups[] = {
 	SND_PCI_QUIRK(0x103c, 0x807C, "HP EliteBook 820 G3", CXT_FIXUP_HP_DOCK),
 	SND_PCI_QUIRK(0x103c, 0x80FD, "HP ProBook 640 G2", CXT_FIXUP_HP_DOCK),
 	SND_PCI_QUIRK(0x103c, 0x828c, "HP EliteBook 840 G4", CXT_FIXUP_HP_DOCK),
+	SND_PCI_QUIRK(0x103c, 0x83b2, "HP EliteBook 840 G5", CXT_FIXUP_HP_DOCK),
 	SND_PCI_QUIRK(0x103c, 0x83b3, "HP EliteBook 830 G5", CXT_FIXUP_HP_DOCK),
 	SND_PCI_QUIRK(0x103c, 0x83d3, "HP ProBook 640 G4", CXT_FIXUP_HP_DOCK),
 	SND_PCI_QUIRK(0x103c, 0x8174, "HP Spectre x360", CXT_FIXUP_HP_SPECTRE),
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1410_8715ba45d0087f3cb67e85819ea5d4fdf7a86c88.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1410_8715ba45d0087f3cb67e85819ea5d4fdf7a86c88.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
